src/Features/SupportConsoleCommands/Commands/MakeCommand.php

```diff
@@ -131,6 +136,41 @@ public function isReservedClassName($name)
         return array_search(strtolower($name), $this->getReservedName()) !== false;
     }
 
+    protected function afterPromptingForMissingArguments(InputInterface $input, OutputInterface $output)
+    {
+        if ($this->didReceiveOptions($input)) {
+            return;
+        }
+
+        if(
+            confirm(
+                label: 'Do you want to make this an inline component?',
+                default: false
+            )
+        )
+        {
+            $input->setOption('inline', true);
+        }
+
+        if(
+            $testSuite = select(
+                label: 'Do you want to create an test?',
```

Comment:
Typo: "an" should be "a". Maybe it's even better to extend it to say `Do you want to create a test file?`, to clarify its functionality a little bit more

Reply:
Thanks! Updated :)